2do_fe/src/components/PopUps/EditTaskPopUp.js

```diff
@@ -15,48 +19,37 @@ function closeTaskDesc() {
   document.getElementById("popUpEditForm").style.display = "none";
 }
 
-export function TaskFormHeader() {
-  return (
-    <header className="TaskForm-header">
-      <p className="FormHeader-text">Edit Task Details</p>
-      <button onClick={closeTaskDesc} className="exit-button">
-        X
-      </button>
-    </header>
-  );
-}
-
-function EditTaskLogo() {
-  return (
-    <center>
-      <div id="logo-container">
-        <div className="AddTask-Logo">
-          <img src={logo} className="AddTask-logo" alt="logo" />
-        </div>
-      </div>
-    </center>
-  );
-}
-
 function TaskDescForm() {
   //use selector to get current task Id
   //const singleTask = useSelector((state) => state.singleTask);
   //dispatch(taskActions.getTaskById(id));
   //console.log(singleTask);
 
-  const dispatch = useDispatch();
-
   const [taskData, setTaskData] = useState({
-    status: "",
+    status: "completed",
```

Comment:
why the status initial state is completed?

Reply:
I was just testing out the edit function and forgot to change it back. Will change it back

Reply:
But this will need to be initialized to get the title, status, desc and id dueDate of the selected task later